dome9CloudBots/bots/storage_account_disable_public_network_access.py

```diff
@@ -0,0 +1,49 @@
+# What it does: Restricts Storage account access to internal subnets only

```

Comment:
please add this documentation to the bots.md file

Reply:
done